mappings/net/minecraft/world/gen/chunk/ChunkNoiseSampler.mapping

```diff
@@ -1,49 +1,111 @@
+	FIELD f_ryartltr horizontalNoiseSize I
+	FIELD f_shocempi wrappedFunctions Ljava/util/Map;
 	FIELD f_tupstqfc firstCellZ I
+	FIELD f_ubegzncu cellStartBlockX I
+	FIELD f_vjvtctpr cellCaches Ljava/util/List;
+	FIELD f_wkuelxms cellStartBlockZ I
+	FIELD f_woxkktfj inCellY I
 	FIELD f_xflyvvkc preliminarySurfaceLevel Lit/unimi/dsi/fastutil/longs/Long2IntMap;
+	FIELD f_xsdqyyrr arrayIndex I
 	FIELD f_ycdqwwen firstNoiseZ I
+	METHOD <init> (IIILnet/minecraft/unmapped/C_fpiyqmft;IILnet/minecraft/unmapped/C_mdkmcezi$C_dsnmcorv;Lnet/minecraft/unmapped/C_lwpfhits;Lnet/minecraft/unmapped/C_jpghdzxc$C_vlyfprqt;Lnet/minecraft/unmapped/C_pfqozslu;)V
+		ARG 1 horizontalSize
+		ARG 2 height
+		ARG 3 minimumY
+		ARG 4 noiseRouter
+		ARG 5 x
+		ARG 6 z
+		ARG 7 structureWeightSampler
+		ARG 8 settings
+		ARG 9 aquiferFluidPicker
+		ARG 10 blender
 	METHOD m_bfhxedds create (Lnet/minecraft/unmapped/C_lwzmmmqr;Lnet/minecraft/unmapped/C_fpiyqmft;Ljava/util/function/Supplier;Lnet/minecraft/unmapped/C_lwpfhits;Lnet/minecraft/unmapped/C_jpghdzxc$C_vlyfprqt;Lnet/minecraft/unmapped/C_pfqozslu;)Lnet/minecraft/unmapped/C_rvjywqnu;
 		ARG 0 chunk
+		ARG 1 noiseRouter
 		ARG 2 noiseFiller
 		ARG 3 settings
 		ARG 4 aquiferFluidPicker
 		ARG 5 blender
 	METHOD m_bjhbvgiq sampleNoiseCorners (II)V
 		ARG 1 noiseY
 		ARG 2 noiseZ
+	METHOD m_bpkwbbvo sampleBlockState ()Lnet/minecraft/unmapped/C_txtbiemp;
 	METHOD m_crjbklqh sampleStartNoise ()V
+	METHOD m_dofytsxx (DLnet/minecraft/unmapped/C_rvjywqnu$C_fzdxentu;)V
+		ARG 2 interpolator
 	METHOD m_enjlfhkh create (IIIILnet/minecraft/unmapped/C_fpiyqmft;Lnet/minecraft/unmapped/C_lwpfhits;Lnet/minecraft/unmapped/C_jpghdzxc$C_vlyfprqt;)Lnet/minecraft/unmapped/C_rvjywqnu;
 		ARG 0 x
 		ARG 1 z
 		ARG 2 minimumY
 		ARG 3 height
+		ARG 4 noiseRouter
 		ARG 5 settings
 		ARG 6 aquiferFluidPicker
+	METHOD m_epzpiuug fillSlice (ZI)V
+		ARG 1 useStartNoiseBuffer
+		ARG 2 x
+	METHOD m_huxgfpyn (IILnet/minecraft/unmapped/C_rvjywqnu$C_fzdxentu;)V
+		ARG 2 interpolator
 	METHOD m_kitscoaj sampleNoiseX (ID)V
 		ARG 2 deltaX
 	METHOD m_kpsxdmhr swapBuffers ()V
+	METHOD m_lxfwjrvk stopInterpolation ()V
 	METHOD m_nqipgozp sampleEndNoise (I)V
 		ARG 1 x
 	METHOD m_nqqcrsfw sampleNoiseY (ID)V
 		ARG 2 deltaY
 	METHOD m_pqgmcdgo sampleNoiseZ (ID)V
 		ARG 2 deltaZ
+	METHOD m_pxlayuiw getWrappedFunction (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 1 function
+	METHOD m_qusmmqcn (DLnet/minecraft/unmapped/C_rvjywqnu$C_fzdxentu;)V
+		ARG 2 interpolator
 	METHOD m_rfsfjiny getAquiferSampler ()Lnet/minecraft/unmapped/C_jpghdzxc;
+	METHOD m_sbkkjxgb (DLnet/minecraft/unmapped/C_rvjywqnu$C_fzdxentu;)V
+		ARG 2 interpolator
 	METHOD m_vjyupqdl getOrComputePreliminarySurfaceLevel (II)I
 		ARG 1 x
 		ARG 2 z
-	METHOD m_wldloquu m_wldloquu (J)I
+	METHOD m_wldloquu computePreliminarySurfaceLevel (J)I
 		ARG 1 chunkPos
+	METHOD m_xbxtgxve getOrComputeBlendingOutput (II)Lnet/minecraft/unmapped/C_pfqozslu$C_yxwnprkr;
```

Comment:
```suggestion
	METHOD m_xbxtgxve computeBlendingOutput (II)Lnet/minecraft/unmapped/C_pfqozslu$C_yxwnprkr;
```
I think this falls under
> Don't name methods based on implementation details. Names should describe what methods do, not how they work.

from [CONVENTIONS:Things to avoid](https://github.com/QuiltMC/quilt-mappings/blob/1.18.2/CONVENTIONS.md#things-to-avoid)

In the case of `getOrComputePreliminarySurfaceLevel` and `computePreliminarySurfaceLevel`, 
I think the former should be `computePreliminarySurfaceLevel` and the latter should be `computePreliminarySurfaceLevelImpl`.

Reply:
`getPreliminarySurfaceLevel` and `computePreliminarySurfaceLevel` would be better imo